docs/upgrade/upgrade-7.17-8.x.asciidoc

```diff
@@ -0,0 +1,158 @@
+.. Ensure that any SOAR workflows that consume alerts are working.
+.. Verify that any custom dashboards your team has created are working properly, especially if they operate on alert (signal) documents.
+
+. If you performed these steps on a non-prod deployment, repeat these same steps on your prod environment. If you're using a cross-cluster search environment and performed these steps on your remote clusters, repeat these same steps on your other deployments. 
+. Confirm with your appropriate stakeholders that the upgrade process is complete.
+
+[float]
+=== Post-upgrade steps
+
+The following sections describe procedures to complete after upgrading {elastic-sec} to 8.x.
+
+[float]
+[[reenable-rules-upgrade]]
+==== Re-enable disabled rules
+
+Any active rules when you upgrade from 7.17 to 8.0.1 or newer are automatically disabled, and a tag named `auto_disabled_8.0` is added to those rules for tracking purposes. Once the upgrade is complete, you can filter rules by the new tag, then use bulk actions to re-enable them:
+
+. Go to the Rules page (*Detect -> Rules*).
+. From the *Tags* dropdown, search for `auto_disabled_8.0`.
+. Click *Select all _x_ rules*, or individually select the rules you want to re-enable.
+. Click *Bulk actions -> Enable* to re-enable the rules.
+
+Alternatively, you can use the <<bulk-actions-rules-api, Bulk rule actions>> API to re-enable rules.
+
+[float]
+[[fda-upgrade]]
+==== Full Disk Access (FDA) approval for {elastic-endpoint}
+
+When you manually install {elastic-endpoint}, you must approve a system extension, kernel extension, and enable Full Disk Access (FDA). There is a new FDA requirement in 8.x. Refer to <<elastic-endpoint-deploy-reqs>> to review the required permissions.
+
+[float]
+[[data-views-upgrade]]
+==== Requirements to display Data views in the {security-app}
+
+To make the *Data view* option appear in an environment with legacy alerts, a user with elevated role privileges must visit the {security-app}, open a page that displays alert data (at least one alert must be present), and then refresh the page. The user's role privileges must allow them to enable the detections feature in a {kib} space. For more information, refer to <<enable-detections-ui, Enable and access detections>>. 
+
+[float]
+[[alert-schema-upgrade]]
+==== New alert schema
+
+The system index for detection alerts has been renamed from `.siem-signals-<space-id>` to `.alerts-security.alerts-<space-id>` and is now a hidden index. Therefore, the schema used for alert documents in {elastic-sec} has changed. Users that access documents in the `.siem-signals` indices using the {elastic-sec} API must modify their API queries and scripts to operate properly on the new 8.x alert documents. Refer to <<query-alert-indices, how to query alert indices>> and review the new <<alert-schema, Alert schema>>.
```

Comment:
[Here](https://github.com/elastic/security-docs/pull/5186/files#diff-c6f4f6d020765a7b8e401186e3a0da3d43ba14059b4a20cfebd5765f44f36b9fR101), "signals" is added in parentheses after "alert". Do you need to do the same in this paragraph for consistency?

Reply:
I could be misremembering, but I vaguely recall "signals" being used synonymously with "alerts" in 7.x and even sometimes in 8.x (e.g., the `max_signals` field allows you to define the max number of alerts a rule can generate). That said, it's probably not necessary to make the change that I suggested. It doesn't add any clarity or necessary context to the explanation about the new alert schema. Feel free to reject this suggestion!

Reply:
We've attempted to avoid/eliminate the use of "signals" altogether in 8.x.  (@nastasha-solomon 's comment about the max_signals field notwithstanding), so can we review all uses of "(signals)" to ensure that we're not persisting its use in 8.x?  Perhaps "(signals in 7.x)" would be more descriptive, or "(formerly signals)" or something similar.